client/src/pages/Rental/Rent.js

```diff
@@ -0,0 +1,274 @@
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+var classNames = require("classnames");
+
+//Mock Data
+
+const CarList = [
+  {
+    name: "Fiat 500",
+    price: "62",
+    total: "125",
+    comment: "Or similar / Sedan",
+    url: "https://drive.google.com/uc?id=1GuQ-LDMfBnkO5SQY5le0wqq2a8OnwEhl",
+  },
+  {
+    name: "Jeep ",
+    price: "25",
+    total: "132",
+    comment: "Or similar / Sedan",
+    url: "https://drive.google.com/uc?id=1COvvFLhKKfSdrthD5vNg9NAq1mPgsJji",
+  },
+  {
+    name: "Peugeot 208",
+    price: "67",
+    total: "136",
+    comment: "Or similar / Sedan",
+    url: "https://drive.google.com/uc?id=1dFv1WeIjdbAGO5eiKYM6TbqDsGGAu4rq",
+  },
+  {
+    name: "Peugeot 308",
+    price: "69",
+    total: "138",
+    comment: "Or similar / Sedan",
+    url: "https://drive.google.com/uc?id=1sxxaGrfKS3S8XUdG9xim_GdwXlKUfujR",
+  },
+  {
+    name: "Honda civic",
+    price: "25",
+    total: "132",
+    comment: "Or similar / Sedan",
+    url: "https://drive.google.com/uc?id=1Feaixf_o87HkyIGYfwupBq6lHl5XmtRk",
+  },
+  {
+    name: "Nissan March Ak12",
+    price: "45",
+    total: "120",
+    comment: "Or similar / Sedan",
+    url: "https://drive.google.com/uc?id=1Bk-DvIXkr2B5BfGe_V6oACKHHwegVVZk",
+  },
+  {
+    name: "Mazda",
+    price: "25",
+    total: "132",
+    comment: "Or similar / Sedan",
+    url: "https://drive.google.com/uc?id=1BwR493KbJ8Nm2lsqttStrxZxPBT2BCL3",
+  },
+  {
+    name: "Nissan pulsar",
+    price: "25",
+    total: "132",
+    comment: "Or similar / Sedan",
+    url: "https://drive.google.com/uc?id=1RNJc5fZTw5Q-yOE_uF4rka6LTT5DekC6",
+  },
+];
+
+const Rent = () => {
+  const classes = useStyles();
+  const [checkState, setCheckState] = useState(false);
+  const handleChange = (event) => {
+    setCheckState(event.target.checked);
+  };
+  return (
+    <Container className={classes.pageContainer}>
+      <Grid container className={classes.filterContainer}>
+        <div className={classes.filterContainer}>
+          <div className={classes.textFieldContainer0}>
+            <TextField
+              id="standard-read-only-input"
+              label="Pick-up"
+              defaultValue="Paris CDG Airport"
+              disabled
+              InputProps={{
+                disableUnderline: true,
+                classes: {
+                  disabled: classes.disabled,
+                },
+              }}
+            />
+          </div>
+          <div className={classes.textFieldContainer1}>
+            <TextField
+              id="standard-read-only-input"
+              label="Return"
+              defaultValue="Paris City Center"
+              disabled
+              InputProps={{
+                disableUnderline: true,
+                classes: {
+                  disabled: classes.disabled,
+                },
+              }}
+            />
+          </div>
+        </div>
```

Comment:
I see you're using a lot of `div`s. Try using more Material UI components instead of plain JSX.